wen-restart/protos/wen_restart.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package solana.wen_restart_proto;
+
+message WenRestartProgress {
+    enum State {
+        INIT = 0;
+        LAST_VOTED_FORK_SLOTS = 1;
+        HEAVIEST_FORK = 2;
+        GENERATING_SNAPSHOT = 3;
+        FINISHED_SNAPSHOT = 4;
+        WAITING_FOR_SUPERMAJORITY = 5;
+        DONE = 6;
+    }
+
+    message InitRecord {
+        uint64 last_vote_slot = 1;
+        string last_vote_bankhash = 2;
+        uint32 shred_version = 3;
+
+    }
+
+    State state = 1;
+    optional InitRecord init_record = 2;
+}
```

Comment:
```suggestion
enum State {
    INIT = 0;
    LAST_VOTED_FORK_SLOTS = 1;
    HEAVIEST_FORK = 2;
    GENERATING_SNAPSHOT = 3;
    FINISHED_SNAPSHOT = 4;
    WAITING_FOR_SUPERMAJORITY = 5;
    DONE = 6;
}

message InitRecord {
    uint64 last_vote_slot = 1;
    string last_vote_bankhash = 2;
    uint32 shred_version = 3;
}

message WenRestartProgress {
    State state = 1;
    optional InitRecord init_record = 2;
}
```
Nit. This is perfectly functional as written; I just didn't see a strong need for State and InitRecord to be wrapped in a submodule.